lib/utils.js

```diff
@@ -51,3 +52,13 @@ export function isSamePath(p1, p2) {
 
   return normalize(p1) == normalize(p2);
 }
+
+/**
+ * @param {string} shellPath
+ * @returns {string[]} - absolute paths of found files
+ */
+export function getFilePathsMatchingPattern(shellPath) {
```

Comment:
I would suggest renaming this function to `globSyncNoEscape`, just because it's shorter and more obvious that it's essentially a `glob`.